packages/dds/sequence/src/defaultMap.ts

```diff
@@ -418,10 +418,7 @@ export class DefaultMap<T> {
 				}
 			},
 			getStashedOpLocalMetadata: (op: IMapValueTypeOperation) => {
-				assert(
-					false,
-					0x016 /* "apply stashed op not implemented for custom value type ops" */,
-				);
+				return op.value as unknown as IIntervalDeltaOp;
```

Comment:
we should generally avoid casting. what is the problem here? can you just fix IMapValueTypeOperation to have the correct typing?